gulpfile.js/util/pattern-library/lib/parseDocumentation.js

```diff
@@ -4,9 +4,17 @@ var nunjucks = require('nunjucks')
 
 var parseDocumentation = function (files) {
   files.forEach(function (file) {
-    nunjucks.configure(path.parse(file.path).dir)
+    nunjucks.configure([
```

Comment:
Should this be in the forEach?

Reply:
Yeah. Nunjucks needs to know the path to each file so that it knows where to look for the markup file.